Assets/Scripts/Backend/Api/UIApi.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// An Api for opening and closing UI windows.
+/// </summary>
+public static class UIApi
+{
+	/// <summary>
+	/// A delegate with List of GameObject parameter.
+	/// </summary>
+	/// <param name="value">The corresponding windows as type GameObject.</param>
+	public delegate void GameObjectListEvent(List<GameObject> value); 
+
+	/// <summary>
+	/// An event for when a window should be opened;
+	/// </summary>
+    public static event GameObjectEvent OnWindowOpen;
+
+	/// <summary>
+	/// An event for when the last opened window should be closed.
+	/// </summary>
+    public static event GameObjectEvent OnCloseLastWindow;
+    
+	/// <summary>
+	/// An event for when all windows should be closed.
+	/// </summary>
+    public static event GameObjectListEvent OnCloseAllWindows;
+
+	/// <summary>
+	/// A List of all opened windows.
+	/// </summary>
+    public static List<GameObject> OpenedWindows = new List<GameObject>();
+
+	/// <summary>
+	/// Opens a window.
```

Comment:
Try to specify summaries more, "Opens a window." Which window does it open? A random window?